scripts/live_test/CLITest.yml

```diff
@@ -166,8 +166,18 @@ jobs:
         pwd
         ls
         # ssh-keygen -f ~/.ssh/id_rsa -t rsa -N ''
-        git clone -b $(USER_BRANCH) $(USER_REPO)
-        #git clone https://github.com/Azure/azure-cli-extensions.git
+        git config --global user.email "AzurePipelines"
+        git config --global user.name "AzurePipelines"
+        # Commit changes
+        if [[ "$(USER_USERNAME)" != "" || "$(USER_TOKEN)" != "" ]]; then
+          REPO="$(USER_REPO)"
+          git clone -b $(USER_BRANCH) ${REPO:0:8}$(USER_USERNAME):$(USER_TOKEN)@${REPO:8}
```

Comment:
Please add some comments about this magical line.

Reply:
What will happen if one of USER_USERNAME and USER_TOKEN is empty?